apps/condo/domains/analytics/constants/constants.js

```diff
@@ -1,5 +1,7 @@
 const EXTERNAL_REPORT_TYPES = ['metabase']
+const EXTERNAL_REPORT_EXPIRATION_SECONDS = 30 * 60
```

Comment:
You use the const just in one place. Probably it's better to move it to the place of usage.